packages/vaadin-grid/test/column-auto-width.test.js

```diff
@@ -113,6 +113,16 @@ describe('column auto-width', function () {
     await recalculateWidths();
     expectColumnWidthsToBeOk(columns, [42, 62, 84, 107]);
   });
+
+  it('should exclude non-visible body cells from grid column auto width calc', async () => {
+    // Assign more items to the grid. The last one with the long content, while in the DOM,
+    // will end up outside the visible viewport and therefor should not affect the
+    // calculated column auto-width
+    grid.items = [...testItems, { a: 'a' }, { a: 'aaaaaaaaaaaaaaaaaaaaa' }];
```

Comment:
Probably one small detail I would like to have clarified: 

Does the long content play a role in the test? Or is it more for the clarity of the example?

Reply:
Yes, it has a purpose in the test. If you remove the changes in the PR, this new test would fail because the cell with the long content would get used as a sample for column auto-width.

Reply:
The content is purposefully long so that if would generate a wider column than with just the default content in the column A:

![Screenshot 2021-05-31 at 16 55 35](https://user-images.githubusercontent.com/1222264/120204107-1f585200-c231-11eb-8a19-d19de0c783a6.png)

Reply:
Perfect explanation, thanks!